python/cuml/test/test_metrics.py

```diff
@@ -1080,3 +1084,46 @@ def test_pairwise_distances_output_types(input_type, output_type, use_global):
             assert isinstance(S, np.ndarray)
         elif output_type == "cupy":
             assert isinstance(S, cp.core.core.ndarray)
+
+
+@pytest.mark.parametrize("input_type, nrows, ncols",
+                         [
+                             unit_param((30, 10, 7)),
+                             quality_param((5000, 100, 50)),
+                             stress_param((500000, 200, 100))
+                         ])
```

Comment:
```suggestion
@pytest.mark.parametrize("input_type, nrows, ncols",
                         [
                             unit_param(30, 10, 7),
                             quality_param(5000, 100, 50),
                             stress_param(500000, 200, 100)
                         ])
```
When using multiple names in `parametrize`, you just want to pass each option as its own arg since the `*args` of `unit_param` are automatically converted to a tuple already. Otherwise you end up with a tuple of tuples.